src/shellintrospect.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright © 2019 Red Hat, Inc
```

Comment:
I suspect there should be less Red Hat here and more you? If anything, the only thing I added was an empty file :P

Reply:
Yes, I think so. Done